test/loading.js

```diff
@@ -50,59 +51,64 @@ async function setTestCase() {
 
 describe('Running Pupeeteer', function() {
   describe('Loading Test Suite', function() {
-    it('Checking Scenes', async (done) => {
+    it('Checking Scenes', async done => {
       let error = false;
       const appTester = new LoadTester({
         slowMo: 0,
         host: 'http://localhost:3000',
```

Comment:
Should use the actual port, which comes from the env.

Reply:
Rectified it.